neqo-transport/src/recovery.rs

```diff
@@ -209,12 +240,173 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
+    congestion_window: u64, // = kInitialWindow
+    bytes_in_flight: u64,
+    congestion_recovery_start_time: Option<Instant>,
+    ssthresh: u64,
+}
+
+impl Default for CongestionControl {
+    fn default() -> Self {
+        CongestionControl {
+            congestion_window: INITIAL_WINDOW,
+            bytes_in_flight: 0,
+            congestion_recovery_start_time: None,
+            ssthresh: u64::max_value(),
+        }
+    }
+}
+
+impl Display for CongestionControl {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "CongCtrl {}/{} ssthresh {}",
+            self.bytes_in_flight, self.congestion_window, self.ssthresh
+        )
+    }
+}
+
+impl CongestionControl {
+    #[cfg(test)]
+    pub fn cwnd(&self) -> u64 {
```

Comment:
Should we be using u64 here or usize?  All of the code that relies on this needs `usize` and it looks like this just forces more conversions (as below).

Reply:
I don't see where it's requiring conversions, can you point one out? I see `cwnd()` being compared to itself at a later time as well as `ssthresh()` and `MIN_CONG_WINDOW` with no conversions needed